.github/workflows/ci.yaml

```
@@ -736,6 +736,12 @@ jobs:
         timeout-minutes: 10
         run: |
```

You may need to force `bash` here.
```suggestion
        shell: bash
        run: |
```

---

Confirmed on https://github.com/home-assistant/core/runs/4739313061?check_suite_focus=true#step:9:31